src/Stripe.net/Services/Subscriptions/SubscriptionCreateOptions.cs

```diff
@@ -25,12 +31,158 @@ public class SubscriptionCreateOptions : SubscriptionSharedOptions
+        /// </summary>
+        [JsonProperty("coupon")]
+        public string Coupon { get; set; }
+
         /// <summary>
         /// REQUIRED: The identifier of the customer to subscribe.
         /// </summary>
         [JsonProperty("customer")]
         public string Customer { get; set; }
 
+        /// <summary>
+        /// Number of days a customer has to pay invoices generated by this subscription. Only valid for subscriptions where <c>billing=send_invoice</c>.
+        /// </summary>
+        [JsonProperty("days_until_due")]
+        public long? DaysUntilDue { get; set; }
+
+        /// <summary>
+        /// ID of the default payment method for the subscription.
+        /// </summary>
+        [JsonProperty("default_payment_method")]
+        public string DefaultPaymentMethod { get; set; }
+
+        [JsonProperty("default_source")]
+        public string DefaultSource { get; set; }
+
+        /// <summary>
+        /// Ids of the tax rates to apply to this subscription.
+        /// </summary>
+        [JsonProperty("default_tax_rates")]
+        public List<string> DefaultTaxRates { get; set; }
+
+        /// <summary>
+        /// List of subscription items, each with an attached plan.
+        /// </summary>
+        [JsonProperty("items")]
+        public List<SubscriptionItemOptions> Items { get; set; }
+
+        /// <summary>
+        /// A set of key/value pairs that you can attach to a subscription object. It can be useful for storing additional information about the subscription in a structured format.
+        /// </summary>
+        [JsonProperty("metadata")]
+        public Dictionary<string, string> Metadata { get; set; }
+
+        /// <summary>
+        /// Indicates if a customer is on session while an invoice payment is attempted.
+        /// </summary>
+        [JsonProperty("off_session")]
+        public bool? OffSession { get; set; }
+
+        /// <summary>
+        /// <para>
+        /// Use <c>allow_incomplete</c> to create subscriptions with <c>status=incomplete</c> if its
+        /// first invoice cannot be paid. Creating subscriptions with this status allows you to
+        /// manage scenarios where additional user actions are needed to pay a subscription's
+        /// invoice. For example, SCA regulation may require 3DS authentication to complete payment.
+        /// See the <a href="https://stripe.com/docs/billing/migration/strong-customer-authentication">SCA Migration Guide</a>
+        /// for Billing to learn more. This is the default behavior.
+        /// </para>
+        /// <para>
+        /// Use <c>error_if_incomplete</c> if you want Stripe to return an HTTP 402 status code if
+        /// a subscription's first invoice cannot be paid. For example, if a payment method requires
+        /// 3DS authentication due to SCA regulation and further user action is needed, this
+        /// parameter does not create a subscription and returns an error instead. This was the
```

Comment:
I tend to editorialize those really long descriptions personally though for autogen it won't work
but the part about the API version is irrelevant to dotnet